packages/gqty/src/Selection/SelectionManager.ts

```diff
@@ -153,36 +115,23 @@ export function createSelectionManager(): SelectionManager {
 
     restoredBackup = backup;
 
-    for (const [aliasKey, alias] of backup[0]) {
-      aliasMap.set(aliasKey, alias);
-    }
-
-    for (const [stringKey, incHashIdValue] of backup[1]) {
-      stringsHash[stringKey] = incHashIdValue;
-      incHashId = incHashIdValue;
-    }
-
-    for (const [key, incId] of backup[2]) {
-      incIds[key] = incId;
+    for (const [stringKey, hashIdValue] of backup[1]) {
```

Comment:
The `backup[1]` looks like a `backup[0]` here, or am I missing something?

Reply:
Good point